doc/sphinx/practical-tools/utilities.rst

```diff
@@ -712,20 +712,20 @@ Building a Coq project with Dune
 .. note::
 
    Dune's Coq support is still experimental; we strongly recommend
-   using Dune 2.3 or later.
+   using Dune 3.1 or later.
 
 .. note::
 
    The canonical documentation for the Coq Dune extension is
    maintained upstream; please refer to the `Dune manual
    <https://dune.readthedocs.io/>`_ for up-to-date information. This
```

Comment:
Let's point to the new Coq page for this, which unfortunately is not available in the stable build of dune, but in the latest. The link should be fine though.

https://dune.readthedocs.io/en/latest/coq.html

Reply:
Sounds good, will amend, I hope it stays stable, as I think we already pointed out to it, then it got removed in Dune!

Problems of doc coordination .

Reply:
Actually it is better just to point to the main Dune page I think.